src/main/java/Duke.java

```diff
@@ -1,10 +1,91 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        final String DIVIDER = "    ——————————————————————————————————————————————————————————————";
```

Comment:
The body of the main function is too long. You need to create separate functions to shorten the length of the code in this function.